install-nicos-client.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- hosts: "{{ host | default('localhost') }}"
+  vars:
```

Comment:
These variables are repetitive! They appear in both install-nicos-client.yml and install-nicos.yml. Putting them in one place makes sense.

This can be done using a `group_vars` folder in the top level directory. See an example here:
https://github.com/ess-dmsc/dm-sinq-amor/blob/master/ansible/group_vars/all